test/unit/services/sailsResponseProcessor_spec.js

```diff
@@ -10,7 +10,12 @@ describe('sailsResponseProcessor', function () {
         id: '1',
         attributes: {
           capacity: 10, availability: 4,
-          expectedins: [{id: 1}, {id: 2}]
+          expectedins: [{id: 1}, {id: 2}],
+          PrebookingDetail: {
```

Comment:
Robin did that in commit: 6cc6a6eccd32b2232caa8fb4b224265ec138a4cd

However, this is not ready for a merge, as I am still working on the API. I produced a PR just for my benefit of seeing what Robin implemented so far. I was under the impression that you were on leave, and thus did not update the tag to 'do not merge'.